test/multi_tenancy.sh

```diff
@@ -175,6 +180,69 @@ function test_user_has_proper_access() {
     fi
 }
 
+## This function tests acces to ES index/alias from kibana pod.
+function test_user_has_proper_access() {
+    local user=$1; shift
+    local pw=$1; shift
+    local alias=$1; shift
+
+    local kpod=$( get_running_pod kibana )
+    local eshost=$( get_es_svc es )
+
+    os::log::info See if user $user can read alias /$alias
+    get_test_user_token $user $pw false
+    #local xfuser='-H x-forwarded-user:'"$user"
+    #local xocpns='-H x-ocp-namespace:'"$user_project_list"
+    #local xfroles='-H x-forwarded-roles:project_user'
+    
+    #nrecs=$( curl_es_from_kibana $kpod $eshost "/$alias/_count" $test_token -XGET | get_count_from_json )
+    #expected=${1:-$user_project_num}
+    #if ! os::cmd::expect_success "test $nrecs = $expected" ; then
+    #    os::log::error $user cannot access alias /$alias
+    #    curl_es_from_kibana $kpod $eshost "/$alias/_count" $test_token -XGET | python -mjson.tool
+    #    oc exec -c elasticsearch $espod -- es_acl get --doc=roles
+    #    oc exec -c elasticsearch $espod -- es_acl get --doc=rolesmapping
+    #    exit 1
+    #fi
+
+    # test user has access for msearch for multiple indices
+    os::log::info See if user $user can _msearch alias "/$alias"
+    nrecs=$( { echo '{"index":'\"${alias}\"'}'; echo '{"size":0,"query":{"match_all":{}}}'; } | \
+                     curl_es_pod_from_kibana_with_token_and_input $kpod $eshost "/_msearch" $test_token -XGET --data-binary @- | \
+                     get_count_from_json_from_search )
+    expected=${1:-$user_project_num}
+    if ! os::cmd::expect_success "test $nrecs = $expected" ; then
+        os::log::error $user cannot access "/$alias" indices with _msearch
+        {
+            echo '{"index":'\"${alias}\"'}';
+            echo '{"size":0,"query":{"match_all" : {}}}';
+        } | curl_es_pod_from_kibana_with_token_and_input $kpod $eshost "/_msearch" $test_token -XGET --data-binary @- 
+        exit 1
+    fi
+
+    # verify normal user has no access to default indices
+    os::log::info See if user $user is denied /project.default.*
```

Comment:
What is the purpose to use the old index here? Same applies to L234-243

Reply:
need to clarify